src/vendor/components/navigation/index.jsx

```diff
@@ -0,0 +1,132 @@
+// eslint-disable react/prop-types
```

Comment:
I would typically avoid putting a HOC component under a `components` directory since it's not really a UI component. Perhaps a `utils` folder.

Reply:
I have no idea the difference between "component", "container", "view", or "hoc" types; they are all "components" to me.  As long as the directory structure is a rough estimate of component dependencies I am happy.  Maybe the name is wrong; I am open to suggestions.

I think the [nimbledroid mess](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/288) was caused by separating component types into directories rather than keeping one component, and its children, together.

Reply:
anyway, you already suggested `utils` which is good enough for me.